app/(voters)/candidate/[slug]/ElectionTab.tsx

```
@@ -1,107 +1,136 @@
-                onMouseEnter={() => setDropdownHovered(true)}
-                onMouseLeave={() => setDropdownHovered(false)}
+    <div className="pl-2 space-y-2">
+      {link.ContentBlock.sort((a, b) => a.order - b.order).map((block) => {
+        const color = block.color ? colorClass[block.color] : "";
+
+        switch (block.type) {
+          case BlockType.HEADING:
+            const headingClass =
+              block.level === 1
+                ? `text-xl font-bold ${color}`
+                : `text-lg font-semibold ${color}`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)